server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/pipelines_controller.rb

```diff
@@ -18,9 +18,10 @@ module ApiV1
   module Admin
     class PipelinesController < ApiV1::BaseController
       before_action :check_admin_user_and_401
-      before_action :load_pipeline, only: [:show]
+      before_action :load_pipeline, only: [:show, :destroy]
```

Comment:
it will raise RecordNotFound Exception if the user tried to delete a pipeline which does not exists